tests/end2end/helpers/E2ESecurity.js

```diff
@@ -0,0 +1,41 @@
+import { E2EGlobal } from './E2EGlobal'
+
+export class E2ESecurity {
+    //Many of the security-e2e-tests will simulate a hacker-attack by calling meteor methods directly.
+    //Thus the names of the called meteor methods are hardcoded in the e2e-Tests and have to be updated if a method is renamed.
+    //In order to check this, all security-e2e-tests should use this function to check if the methods called within them do still exist. 
```

Comment:
Instead of trust the developer to use this function before executing the meteor method (via `E2ESecurity.executeMethod`) we could force calling this function by applying it in the `E2ESecurity.executeMethod`?